impeller/renderer/backend/metal/allocator_mtl.h

```diff
@@ -6,18 +6,39 @@
 #define FLUTTER_IMPELLER_RENDERER_BACKEND_METAL_ALLOCATOR_MTL_H_
 
 #include <Metal/Metal.h>
+#include <atomic>
 
+#include "impeller/base/thread.h"
 #include "impeller/core/allocator.h"
 
 namespace impeller {
 
+class DebugAllocatorStats {
+ public:
+  DebugAllocatorStats() {}
+
+  ~DebugAllocatorStats() {}
+
+  void Increment(size_t size);
```

Comment:
nit: note the units of `size` in a docstring.

Reply:
Done